.github/workflows/android-master.yml

```diff
@@ -30,29 +30,30 @@ jobs:
       - name: Unit Tests
         run: bash ./gradlew test --stacktrace
 
-    aar:
-      name: Generate AAR
-      # The type of runner that the job will run on
-      runs-on: ubuntu-20.04
-
-      steps:
-        # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
-        - uses: actions/checkout@v2
-
-        # Sets up a JDK on the job so that we can run Java code
-        - name: Set up JDK 1.8
-          uses: actions/setup-java@v1
-          with:
-              java-version: 1.8
-
-        # Call Gradle to build our our debug AAR
-        - name: Build debug AAR
-          run: bash ./gradlew assembleDebug --stacktrace
-
-        # Stores the generated AAR to an outputs folder
-        - name: Store AAR
-          uses: actions/upload-artifact@v2
-          with:
-            name: sdk
-            path: app/build/outputs/aar/debug/aar-debug.aar
+  # A second job for building our our SDK's AAR file begins here
+  aar:
```

Comment:
Nothing changed except for indentation to make this be picked up as a job, and the addition of a comment on the line above to highlight its purpose